syndicate/core/resources/lambda_resource.py

```diff
@@ -842,6 +842,7 @@ def _create_sqs_trigger_from_meta(self, lambda_name, lambda_arn, role_name,
                                       trigger_meta):
         validate_params(lambda_name, trigger_meta, SQS_TRIGGER_REQUIRED_PARAMS)
         target_queue = trigger_meta['target_queue']
+        enable_batch_report_failures = trigger_meta.get("enable_batch_report_failures", False)
```

Comment:
Please stick with lines of code no longer than 80 symbols

Reply:
I moved method parameters to new line